app/scripts/services/bindingModalUtils.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .factory('BindingModalUtils', [
```

Comment:
Is this only used in the bind modal and no longer needed for unbind? I suggest for now we leave the working bind code alone if we're not reusing it for unbind.